tools/openvino_dev/CMakeLists.txt

```diff
@@ -101,4 +101,5 @@ ov_cpack_add_component(${OV_CPACK_COMP_PYTHON_WHEELS} HIDDEN)
 install(FILES ${openvino_wheel_path}
         DESTINATION ${OV_CPACK_WHEELSDIR}
         COMPONENT ${OV_CPACK_COMP_PYTHON_WHEELS}
-        ${OV_CPACK_COMP_PYTHON_WHEELS_EXCLUDE_ALL})
+        ${OV_CPACK_COMP_PYTHON_WHEELS_EXCLUDE_ALL}
+        OPTIONAL)
```

Comment:
it will not be required if Python wheel is built via Developer package

Reply:
Yes, but in case if you build ie_wheel target only and try to install it the cmake install throws an error due to absent openvino-dev wheel. So I would suggest to make it optional until we drop it at all

Reply:
just use OpenVINODeveloperPackage approach